docs/dev-setup-local.md

```diff
@@ -415,3 +455,55 @@ You should also be able to send messages with `4`. If there are multiple clients
 connected, just kill and restart the simulator, then wait for the Nuntium SMPP
 service to reconnect (a few seconds) so there is only one (we can easily end up
 with multiple clients connected after editing a channel for example).
+
+## Troubleshooting
+
+### lvh DNS error
+If you are facing a DNS error when navigating to any `lvh.me` domain maybe the problem is your ISP's DNS.
+Try adding public DNS servers to your network configuration. For example:
+- Google:
+  - 8.8.8.8
+  - 8.8.4.4
+- Cloudflare:
+  - 1.1.1.1
+  - 1.0.0.1
+
+More public [DNS servers](https://www.lifewire.com/free-and-public-dns-servers-2626062).
+
+### Bad Gateway error
+If the browser returns a `502 Bad Gateway` error when trying to navigate to `http://web.verboice.lvh.me`, `http://web.nuntium.lvh.me`, `http://app.surveda.lvh.me` (or any other app using a `*.lvh.me` domain) then the error could be related to the app's container not being added to the `shared` network created by `dockerdev`.
+
+First, let's see if this is actually the error.
+For example, if we can't browse `Verboice`, in a terminal run:
+```console
+$ docker inspect verboice_web_1 | jq '.[].NetworkSettings.Networks | keys'
+```
+If this command returns something like this:
+```console
+[
+  "verboice_default"
+]
+```
+without `shared` then we need to add the container to this network. For this, we must destroy and recreate the container until it's properly added to the `shared` network:
+
+```console
+$ docker kill verboice_web_1
+$ docker container rm verboice_web_1
+$ docker-compose up web
```

Comment:
We can use `docker-compose` to destroy a container:
```suggestion
```console
$ docker-compose down web
$ docker-compose up web
```